src/main/antlr/OpenDistroSqlParser.g4

```diff
@@ -324,20 +336,29 @@ aggregateWindowedFunction
     | COUNT '(' aggregator=DISTINCT functionArgs ')'
     ;
 
+functionAsAggregatorFunction
+    : (AVG | MAX | MIN | SUM)
+      '(' aggregator=(ALL | DISTINCT)? functionCall ')'
+    ;
+
 scalarFunctionName
     : functionNameBase
     ;
 
 functionArgs
-    : (constant | fullColumnName | functionCall | expression)
+    : (constant | fullColumnName | expression)
```

Comment:
why remove the functionCall from args?

Reply:
`functionCall` is moved to `nestedFunctionArgs`.